pages/cart.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>navbar | fastcart</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="author" content="Mohasin Haque">
+    <meta name="description" content="Responsive navbar for e commerce sites.">
+    <link rel="stylesheet" href="../styles/main.css">
+    <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+</head>
+
+<body>
+    <nav class="navbar-ecom">
+        <a href="../index.html" class="brand-title-ecom">fastcart</a>
+        <a href="#" class="toggle-button-ecom">
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
```

Comment:
Why an empty span?

Reply:
I think that's for hamburger icon. It will be better to use a readymade icon instead.